### ISSUE TITLE
Accessibility (major): Landing page missing discernible text [prea-004] 

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/pre-assembly#1259]
## Description

There seems to be an extraneous empty `<a>` in the main nav bar, perhaps serving as visual padding for the main Stanford Libraries link.

```<a class="navbar-brand" href="#"></a>```

Screen readers announce this link and read the URL. If this link serves a function, then it should have informative link text. If not, then it should be removed, and padding can be added to the Stanford Libraries link with CSS.

https://docs.google.com/spreadsheets/d/105GozrKuWrnXaEk5FoP4UbzHSCTPgeEqPaPiHi1c950/edit#gid=0

**Comment by lwrubel** [comment]
I think this link is intended to go to the top-level Preassembly URL. The image being for the Libraries is confusing here. (See question on https://github.com/sul-dlss/pre-assembly/pull/1262). 

Either we use a different image and use an informative link about it being Preassembly Home, or make this go to the Libraries home page and add a link somewhere else to Preassembly Home.

**Comment by andrewjbtw** [comment]
Is this still blocked by a question or did the change in #1262 clear that up?

**Comment by edsu** [comment]
@aaron-collier will take another look given the work that has been done in the app header and see if this is still an issue.

**Comment by aaron-collier** [comment]
Yes, this appears complete.
